lib/ansible/module_utils/k8s/raw.py

```diff
@@ -29,8 +29,6 @@
 from ansible.module_utils.k8s.common import KubernetesAnsibleModule
 from ansible.module_utils.common.dict_transformations import dict_merge
 
-from distutils.version import LooseVersion
```

Comment:
Do we not need this? It looks like it's still used below

Reply:
See line 23